app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +29,14 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        findViewById(R.id.button).setOnClickListener( new OnClickListener() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/8d3333d0ff432d0557742aed5ff0615effcedf8c/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L33"></span>’(‘ is followed by whitespace.</td>
    </tr>
  </tbody>
</table>